django-verdant/rca/models.py

```diff
@@ -821,35 +806,26 @@ class ProgrammePageAd(Orderable):
         SnippetChooserPanel('ad'),
     ]
 
-
-class ProgrammePageProgramme(models.Model):
-    page = ParentalKey('rca.ProgrammePage', related_name='programmes')
-    programme = models.ForeignKey('taxonomy.Programme', null=True, on_delete=models.SET_NULL, related_name='programme_pages', help_text=help_text('rca.ProgrammePageProgramme', 'programme'))
-
-    panels = [FieldPanel('programme')]
-
-
 class ProgrammePage(Page, SocialFields, SidebarBehaviourFields):
-    school = models.ForeignKey('taxonomy.School', null=True, on_delete=models.SET_NULL, related_name='programme_pages', help_text=help_text('rca.ProgrammePage', 'school'))
+    school = models.ForeignKey('taxonomy.School', null=True, on_delete=models.SET_NULL, related_name='programme_pages')
```

Comment:
We should keep the ``help_text`` attribute as this makes it easier for us to add help text later.